src/coreclr/jit/objectalloc.cpp

```diff
@@ -673,6 +673,9 @@ unsigned int ObjectAllocator::MorphAllocObjNodeIntoStackAlloc(
         if (predBlock->hasProfileWeight())
         {
             block->setBBProfileWeight(predBlock->bbWeight);
+            JITDUMP("Profile weight into " FMT_BB " needs to be propagated to successors. Profile %s inconsistent.\n",
+                    block->bbNum, comp->fgPgoConsistent ? "is now" : "was already");
+            comp->fgPgoConsistent = false;
```

Comment:
Ah, I should try harder to fix this one up. With PGO in most cases this path will have zero weight.

Reply:
In my experience, this path triggered consistency checks pretty rarely, and I expect that we'll want to run profile repair after the first flowgraph simplification pass for now, so this might fix itself trivially.